### PR TITLE
[Docs] Refresh `mlrun_jobs` example

### PR DESCRIPTION
**Pull request by jillnogold**

### CODE REVIEW

File: docs/runtimes/mlrun_jobs.ipynb [hunk 1]
```diff
@@ -955,19 +926,19 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "Pipeline results are stored at the `artifact_path` location:"
+    "Pipeline results are stored at the `artifact_path` location. By default, MLRun adds ```/{{workflow.uid}}``` to the path."
```
Comment (theSaarco) [inline docs/runtimes/mlrun_jobs.ipynb:929]: It's a bit different than that - the artifact path for workflows can be either:
1. The artifact_path as passed to the specific call for `run()`, as shown below. In this case, MLRun does not modify the user-provided path
2. The project's artifact_path (set by `project.spec.artifact_path = '<some path>'`)
3. MLRun's default artifact-path, if set.

In the 2nd and 3rd option MLRun will add `/{{workflow.uid}}` to the path as described, but only if the path does not already contain it.

**Review by theSaarco — CHANGES_REQUESTED** [review]
Looks good, I think the artifact-path section needs some additional explanation, see in the comments.

**Review by theSaarco — APPROVED** [review]
LGTM

**Comment by Hedingber** [comment]
We want to discourage users from using the `%nuclio` magic commands so I think we shouldn't even specify it

**Comment by jillnogold** [comment]
@theSaarco Please advise on above, how to adjust content for removing nuclio magic

**Comment by theSaarco** [comment]
@jillnogold - we kept the `%nuclio` reference just as another alternative to specifying requirements, for completeness. I am not overly attached to it, and as @Hedingber mentioned, there is already a better way that we describe here. 
So, I'm ok with removing the `%nuclio` reference here - let's just say that you need to pass requirements through the `code_to_function` API and use the example we already added.

**Pull request merged**